src/Infrastructure/LeanCode.Firebase.FCM/FCMClient.cs

```diff
@@ -60,6 +62,25 @@ public async Task SendToUserAsync(Guid userId, MulticastMessage message, bool dr
             }
         }
 
+        public async Task SendToUsersAsync(List<Guid> userIds, MulticastMessage message, bool dryRun, CancellationToken cancellationToken = default)
+        {
+            var tokens = await tokenStore.GetTokensAsync(userIds, cancellationToken);
+            message.Tokens = tokens.SelectMany(t => t.Value).ToList();
+
+            if (message.Tokens.Count == 0)
+            {
+                logger.Information("Cannot send push to user {UserId} - no tokens", userIds);
```

Comment:
`userIds` is a collection while the template implies that only one user per message will be skipped.